### DISCUSSION TITLE
Various addons not working, how do I fix this?

### DISCUSSION THREAD

**Discussion by jayrattt** [ScratchAddons/ScratchAddons#2003] [Help]
Hey, everyone.
I've just updated to Scratch Addons v1.12.0, but a few addons I have enabled are **no longer working**, and I have no idea how to fix it.
The following are screenshots of the broken addons.
_The following addons are supposed to be active here:_
_Live Featured Project, [My-Ocular Status](https://shefwerld.rirurin.com/post/user?user=8bitjake), and Profile Page Banner (you can see the banner shape but there's nothing there)._
![no banner or feature](https://user-images.githubusercontent.com/75950907/112875317-a8180d80-9089-11eb-976b-ee92add9abe7.PNG)
_The **Pause Button** is also missing._
![image](https://user-images.githubusercontent.com/75950907/112875827-4efca980-908a-11eb-9559-293f1ec7ed54.png)
And finally, the statistics are missing as well.
![image](https://user-images.githubusercontent.com/75950907/112875948-79e6fd80-908a-11eb-8f9d-639875bd6153.png)
Again, no idea why this is happening. And, this seems to be because of the update, everything was working fine before I updated.
If you can help me fix this, that's great! But until then, I'm downgrading to the previous version.

_Edit: The Remix Tree button is also missing:_
![image](https://user-images.githubusercontent.com/75950907/112876396-19a48b80-908b-11eb-9512-b362e6aeb5f5.png)
_Edit **AGAIN**: Apparently, every single addon that involves (blank)+clicking the green flag also doesn't work. I forgot to say this, but I'll be editing this post whenever I find a new broken addon._

**Comment by mxmou** [comment]
Do you have multiple versions of Scratch Addons installed at once?

**Reply by jayrattt** [reply]
Nope.

**Comment by TheColaber** [comment]
Could you open the console (Ctrl + Shift + J) and screenshot all the errors on the console?

**Reply by jayrattt** [reply]
Alright, here's everything. Sorry it took me so long to respond, I don't go on Github very often, if at all.
Oh, and I edited the reply because I realized what the errors were and screenshot them instead of everything else.
![image](https://user-images.githubusercontent.com/75950907/112898523-8a0cd600-90a6-11eb-921f-126c1decc570.png)
![image](https://user-images.githubusercontent.com/75950907/112898459-79f4f680-90a6-11eb-8b71-8b1f3a22ae74.png)
![image](https://user-images.githubusercontent.com/75950907/112898617-aad52b80-90a6-11eb-8a93-1b25f464bff3.png)

**Reply by WorldLanguages** [reply]
All those errors are normal, do you have a discord account?

**Reply by jayrattt** [reply]
Yes, I do. Why do you ask?

**Reply by WorldLanguages** [reply]
@8bitjake Can you join this server so that we can help you out find the issue https://discord.gg/2qKGg5gGAh
